tests/kafka_with_topic.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { test } from '@playwright/test';
+import login from '@lib/auth';
+import { config } from '@lib/config';
+import { navigateToKafkaList } from '@lib/kafka';
+
+const testInstanceName = `test-instance-${config.sessionID}`;
```

Comment:
is this a left over?

Reply:
no, everything is used

Reply:
I did not mean that we need that file right away, that was just to have a system in place and once the file is needed, it can be created with some new tests.

Reply:
ok I remove it as well